engine/src/main/java/com/datatorrent/stram/engine/OperatorContext.java

```diff
@@ -84,12 +87,13 @@ public void setIdleTimeout(long idleTimeout)
    * @param attributes the value of attributes
    * @param parentContext
    */
-  public OperatorContext(int id, AttributeMap attributes, Context parentContext)
+  public OperatorContext(int id, String operatorName, AttributeMap attributes, Context parentContext)
```

Comment:
A lot of tests create OperatorContext instance without OperatorDeployInfo. 
OperatorDeployInfo does not expose methods to set the fields - id, name, attributes.
So if we change this it will cause lot more changes.